client/packages/programs/src/JsonForms/components/EnrolmentId.tsx

```diff
@@ -0,0 +1,73 @@
+import { ControlProps, rankWith, uiTypeIs } from '@jsonforms/core';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { DetailInputWithLabelRow } from '@openmsupply-client/common';
+import React from 'react';
+import { z } from 'zod';
+import { useProgramEnrolments } from '../../api';
+import {
+  DefaultFormRowSpacing,
+  FORM_LABEL_WIDTH,
+  useZodOptionsValidation,
+} from '../common';
+
+const Options = z
+  .object({
+    programEnrolmentType: z.string(),
+  })
+  .strict();
+type Options = z.infer<typeof Options>;
+
+const UIComponent = (props: ControlProps) => {
+  const { label, uischema, config } = props;
+
+  const { errors, options } = useZodOptionsValidation(
+    Options,
+    uischema.options
+  );
+
+  // fetch current encounter
```

Comment:
Need to update comment here I think -- "fetch enrolments"?

Reply:
Thanks done!